### ISSUE TITLE
In-chat help issues

### ISSUE THREAD

**Issue by LawOfTheLand** [RolePlayGateway/roleplaygateway#48]
Typing in `/help command` to learn more about the command only ends up reposting the wall of text that is the `/help` prompt.

**Comment by lostamongtrees** [comment]
can confirm, tried it a whole bunch of different ways 

The `/help` menu also needs an update in regards to some nonfunctioning commands, unless they will become functioning with 2.0 . `/ic character name` does not work, for example.
